js/nested-for-loops.js

```diff
@@ -5,11 +5,25 @@
  * @returns {string}
  */
 function nestedForLoops (depth, width = depth) {
-  let result = "\n";
+  let result = "\n"; // "NEW LINE"
 
-  // Write code here
+  let arr = [];
+
+  for(a=0; a<depth; a++){
+    arr[a] = [];
+    for(b=0; b<width; b++){
+      arr[a][b] = '{x:' + b + ', y:' + a + '}';
+      if(b<width && b>0){
+        arr[a][b] = ' ' + arr[a][b];
+      }
+      arr.join(',');
+      
+    }
+  };
+  return result + arr.join(result) + result;
```

Comment:
I'm not sure if this will pass the test, since the inner arrays aren't going to be joined and should print like an array rather than a full string.